playback_descriptions.md

```diff
@@ -0,0 +1,263 @@
+# Playback Filter Descriptions
```

Comment:
This seems to be the same file as docs/playback_filters.md. Should it be removed from this PR?